manifests/postgresql-operator-default-configuration.yaml

```diff
@@ -23,6 +23,7 @@ configuration:
     secret_name_template: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
     oauth_token_secret_name: postgresql-operator
     pod_role_label: spilo-role
+    # spilo_fsgroup: "103"
```

Comment:
I think this should be without quotes, otherwise `readUint64` would complain.

Reply:
Removed the quotes!